### PR TITLE
Created a route to check what emails are already connected to an identity

### PR DESCRIPTION
**Pull request by NateTheDev1**
# Description of Changes

Created a route to check what emails are already connected to an identity. Mainly for the web to not continue to prompt users to connect an email.

Example: GET /:identity/emails

# API and ABI

 - [ ] This is a breaking change to the module ABI
 - [ ] This is a breaking change to the module API
 - [x] This is a breaking change to the ClientAPI
 - [ ] This is a breaking change to the SDK API

*If the API is breaking, please state below what will break*

# Expected complexity level and risk

Rating: 1

*How complicated do you think these changes are? Grade on a scale from 1 to 5,
where 1 is a trivial change, and 5 is a deep-reaching and complex change.*

*This complexity rating applies not only to the complexity apparent in the diff,
but also to its interactions with existing and future code.*

*If you answered more than a 2, explain what is complex about the PR,
and what other components it interacts with in potentially concerning ways.*

### CODE REVIEW

File: crates/core/src/control_db.rs [hunk 1]
```diff
@@ -291,6 +291,19 @@ impl ControlDb {
         Ok(result)
     }
 
+    pub fn get_emails_for_identity(&self, identity: Identity) -> Result<Vec<IdentityEmail>> {
+        let mut result = Vec::<IdentityEmail>::new();
+        let tree = self.db.open_tree("email")?;
+        for i in tree.iter() {
```
[the remark this answers is not in the document]
Comment (kim) [reply crates/core/src/control_db.rs:297]: Wait, was the conclusion not that is should be exactly one email?

**Comment by kim** [comment]
Please note that this is a breaking change without a corresponding patch in cloud.

File: crates/core/src/control_db.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (NateTheDev1) [reply crates/core/src/control_db.rs:297]: Yeah Tyler confirmed that it should work as 1 identity to many emails

[the remark this answers is not in the document]
Comment (kim) [reply crates/core/src/control_db.rs:297]: I'm confused. But okay, that'll make things easier :)

**Review by Centril — APPROVED** [review]
Code itself looks good, but could do with a minor refactoring

**Pull request merged**